doc/classes/Array.xml

```diff
@@ -17,14 +17,14 @@
 		print(array[-3]) # Prints "Second"
 		[/gdscript]
 		[csharp]
-		var array = new Godot.Collections.Array{"First", 2, 3, "Last"};
+		Godot.Collections.Array array = ["First", 2, 3, "Last"];
 		GD.Print(array[0]); // Prints "First"
 		GD.Print(array[2]); // Prints 3
-		GD.Print(array[array.Count - 1]); // Prints "Last"
+		GD.Print(array[^1]); // Prints "Last"
```

Comment:
Is this common knowledge? As a not particularly savvy C# developer, I don't find this shorthand to be more understandable at a glance. Of course, code style is up to you though.

Reply:
I would say so, it's at the very least equivalent to GDScript's negative indexes, which I don't find any easier to understand.

But I imagine the reason why the GDScript example uses negative indexes is to teach users that they can use them to access elements from the end of the array. In a similar way, we can use the C# example to teach them how to do the same thing in C#. So it probably doesn't matter if they're not familiar with this syntax.